helpers/osinfo.go

```diff
@@ -226,7 +226,9 @@ func (btfi *OSInfo) discoverOSDistro() error {
 		}
 		btfi.osReleaseFieldValues[keyID] = val[1]
 		if keyID == OS_ID {
-			btfi.osReleaseID = stringToOSReleaseID[strings.ToLower(val[1])]
+			osVal := strings.ToLower(val[1])
+			osVal = strings.Trim(osVal, "\"") // trim potential parentheses (in centos)
```

Comment:
quotes?

Reply:
?

Reply:
![image](https://github.com/aquasecurity/libbpfgo/assets/3372117/46596f42-1e84-4e08-a333-6f05f509bda9)

Reply:
You mean the comment should be
`// trim potential quotes (in centos`?

Reply:
Done.